From df162f9168205d73a7f43de57388dd70e923d4af Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 18 Mar 2019 17:17:34 +0000
Subject: [PATCH] TOOLS-2101 jsstyle submodules too old for old perls Reviewed
 by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 .gitmodules  | 2 +-
 deps/jsstyle | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 0de3bcf..36d60ff 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,6 +1,6 @@
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = https://github.com/davepacheco/jsstyle.git
+	url = https://github.com/joyent/jsstyle.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
 	url = https://github.com/davepacheco/javascriptlint.git
diff --git a/deps/jsstyle b/deps/jsstyle
index da42b50..52dc973 160000
--- a/deps/jsstyle
+++ b/deps/jsstyle
@@ -1 +1 @@
-Subproject commit da42b50ceb12d431437b32efd4c411a8e2fac0c8
+Subproject commit 52dc973cf64da11834eca7cf46ebce8518e3ee88
-- 
2.21.0

